docs/src/global/code-theme.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css?family=Source+Code+Pro:500');
+@import 'prismjs/plugins/command-line/prism-command-line.css';
+
```

Comment:
@sweeting can you help me develop the proper theme colors. 

i think we can use this tool to create the theme https://k88hudson.github.io/syntax-highlighting-theme-generator/www/

the current docs only have a dark theme...not sure if we want to keep that as is